internal/ignition/ignition.go

```diff
@@ -97,6 +99,11 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 	}
 	installConfigPath := filepath.Join(g.workDir, "install-config.yaml")
 
+	g.enableMetal3Provisioning, err = common.VersionGreaterOrEqual(g.cluster.Cluster.OpenshiftVersion, "4.7")
```

Comment:
I am not sure if we support older versions (will this check above be required).

Reply:
only 4.6 and 4.7

Reply:
this logic is irrelevant to anything before 4.7, no?